### DISCUSSION TITLE
🔗 Add Support for Post and Product Gallery Dynamic Tags/Content in the Loop Grid

### DISCUSSION THREAD

**Discussion by threeforces** [elementor/elementor#23737] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Background:
We run an online shop with multiple WooCommerce products. On select pages, we showcase a summary of specific products via Loop-Grid or Loop-Carousel. Presently, it's possible to attach the static Product-Image to a container via a dynamic attribute for background purposes.

Issue:
However, there's no straightforward way to integrate product galleries into these loops in a similar dynamic manner.

### Describe the solution you'd like

- Allow the integration of product galleries as dynamically as single product images.
- Provide an option to set the product gallery images as the background of a container.
- While product image and category image exist as dynamic attributes, adding support for product gallery would elevate the dynamic feel of the loops.

### Describe alternatives you've considered

Add an additional dynamic widget or Product Gallery like Post-Image

### Additional context

![image](https://github.com/elementor/elementor/assets/54294153/1a1002ea-3755-49a5-87da-2479f5213fdc)

![image](https://github.com/elementor/elementor/assets/54294153/bc644233-402e-4213-9aad-8eec9f17563f)

This addition would enhance the user experience by offering more visual insights into our products and make our product loops more dynamic and visually appealing.

**Comment by awebguy99** [comment]
We need this. It would let us do things like this in loop grid items;
1. Display the main product image as default.
2. Then when a user hovers over the item, it changes to a product gallery image (for that product obviously).

A lot of good e-commerce stores have this.
